Sources/PointFree/Subscribe.swift

```diff
@@ -54,7 +54,7 @@ private func subscribe(_ conn: Conn<StatusLineOpen, Tuple2<SubscribeData, User>>
           let parallel = sequence(
             subscribeData.teammates
               .filter { email in email.rawValue.contains("@") && email != user.email }
-              .prefix(subscribeData.pricing.quantity - 1)
+              .prefix(subscribeData.pricing.quantity - (subscribeData.isOwnerTakingSeat ? 1 : 0))
```

Comment:
we need to truncate this data differently depending on whether or not the owner is taking a seat. a test actually caught this which is nice.